content/common/frame_messages.h

```diff
@@ -878,6 +878,12 @@ IPC_MESSAGE_ROUTED2(FrameMsg_AddMessageToConsole,
                     content::ConsoleMessageLevel /* level */,
                     std::string /* message */)
 
+// Request for the renderer to insert user stylesheet.
+//
+// stylesheet is user stylesheet string
```

Comment:
Would be good to add a reference here for the issue/ticket

Reply:
ACK